plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -103,19 +107,21 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject,
     if (libraries.isEmpty()) {
       return;
     }
-    SubMonitor subMonitor = SubMonitor.convert(monitor, "Adding App Engine libraries", libraries.size());
+    SubMonitor subMonitor = SubMonitor.convert(monitor,
+        Messages.getString("adding.appengine.libraries"), libraries.size()); //$NON-NLS-1$
```

Comment:
Either this should be `adding.app.engine.libraries` (dot between `app` and `engine`), or that dot should be removed from `messages.properties`.